android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -1372,10 +1356,11 @@ public void performPreInflationStartup() {
         BraveDbUtil dbUtil = BraveDbUtil.getInstance();
         if (dbUtil.dbOperationRequested()) {
             AlertDialog dialog = new AlertDialog.Builder(this)
-            .setMessage(dbUtil.performDbExportOnStart() ? "Exporting database, please wait..."
-                        : "Importing database, please wait...")
-            .setCancelable(false)
-            .create();
+                                         .setMessage(dbUtil.performDbExportOnStart()
+                                                         ? "Exporting database, please wait..."
```

Comment:
nit: Better to put message/label strings in resources

Reply:
done